server/php/UploadHandler.php

```diff
@@ -132,18 +132,13 @@ public function __construct($options = null, $initialize = true, $error_messages
             'identify_bin' => 'identify',
             'image_versions' => array(
                 // The empty image version key defines options for the original image:
-                // Keep in mind that these options are inherited by all other image versions!
+                // Warning: these image manipulations are inherited by all other image versions! 
+                // Keep in mind that this does not apply to 'no_cache'.
                 '' => array(
                     // Automatically rotate images based on EXIF meta data:
                     'auto_orient' => true
                 ),
-                // Uncomment the following to create medium sized images:
-                /*
-                'medium' => array(
-                    'max_width' => 800,
-                    'max_height' => 600
-                ),
-                */
```

Comment:
I would rather keep those comment lines, as it shows users how to add different image versions.